### DISCUSSION TITLE
CondaHTTPError: HTTP 000 CONNECTION FAILED

### DISCUSSION THREAD

**Discussion by psaitas** [idaholab/moose#22872] [Q&A General] [answered]
Hello guys I am trying to install moose and i reached the point where is recommended is deactivate and activate again and I obtain the following:

Collecting package metadata (current_repodata.json): failed

CondaHTTPError: HTTP 000 CONNECTION FAILED for url <https://conda.software.inl.gov/public/linux-64/current_repodata.json>
Elapsed: -

An HTTP error occurred when trying to retrieve this URL.
HTTP errors are often intermittent, and a simple retry will get you on your way.
'https//conda.software.inl.gov/public/linux-64'

**Comment by milljm** [comment, marked as answer]
The connection is up and functioning as far as I can tell. Are you able to brows to that link using your web browser?
 
https://conda.software.inl.gov/public/linux-64/current_repodata.json

If that works, can you try using curl on the command line? Please copy and paste the results if you can't. The verbose argument might shed some additional light on the issue.

```bash
curl -v https://conda.software.inl.gov/public/linux-64/current_repodata.json
```

**Comment by psaitas** [comment]
I ran it ang i got the following:

* Connection #0 to host conda.software.inl.gov left intact

On Mon, Dec 5, 2022 at 3:02 PM Jason Miller ***@***.***>
wrote:

> The connection is up and functioning as far as I can tell. Are you able to
> brows to that link using your web browser?
>
> https://conda.software.inl.gov/public/linux-64/current_repodata.json
>
> If that works, can you try using curl on the command line? Please copy and
> paste the results if you can't. The verbose argument might shed some
> additional light on the issue.
>
> curl -v https://conda.software.inl.gov/public/linux-64/current_repodata.json
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/idaholab/moose/discussions/22872#discussioncomment-4313518>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AYSZOIOCUWMCLJWNQF3MND3WLXYWLANCNFSM6AAAAAASUJZNY4>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 

Με εκτίμηση,



*Ηλίας Ψιλάκης*

*Διπλ. Μηχανολόγος & Αεροναυπηγός Μηχανικός*



Τ: +30 69 843 90 843

E: ***@***.***



ü Please consider the environment before printing this e-mail.

**Reply by milljm** [reply]
Your organization might be blocking access to this server. I'm afraid there is nothing we can do to in this case.

**Reply by GiudGiud** [reply]
You ll need to contact your network or cluster administrator
